internal/commands/list.go

```diff
@@ -23,13 +24,24 @@ func List(c *cli.Context) error {
 
 	w := writer.NewTSVWriter()
 
-	header := []string{"ID", "Task", "RemindTime", "reminder"}
+	header := []string{"ID", "Task", "RemindTime", "reminder", "Priority"}
 	if err := w.Write(header); err != nil {
 		return err
 	}
 
-	for _, t := range h.GetTasks() {
-		if err := w.Write([]string{strconv.Itoa(t.ID()), t.Task(), string(t.RemindTime()), string(t.Reminder())}); err != nil {
+	tasks := h.GetTasks()
+	sort.SliceStable(tasks, func(i, j int) bool {
+		switch c.String("order") {
+		case "priority":
+			return tasks[i].Priority() < tasks[j].Priority()
```

Comment:
[imo]
This business rule should be in `internal/entities/task/task_handler_test.go`. That makes more testable.

Reply:
I moved sorting function to `internal/entities/task/task_handler.go`
@ 3f79c61